controllers/maker-controller.js

```diff
@@ -31,11 +31,8 @@ Router.use(async (req, res, next) => {
       req.app = app;
     }
 
-    if (apps.length) {
-      req.apps = apps;
-      return next();
-    }
-    return res.status(400).json({ success: false });
+    req.apps = apps;
```

Comment:
@kyranjamie I think you mentioned we should do this already, right? But I didn't see any code for it anywhere. Anyways, I saw that the recent change we made to pre-fetch maker apps on the server was causing 500s if you didn't have any apps yet. It's because the call would 400. We should wrap that logic in a try/catch, but I also added this because it makes more sense to return an empty array vs. 400.

Reply:
Yep thanks, I never pushed the change.